README.md

```
@@ -257,100 +415,164 @@ npx playwright test --ui
+```
 
 ---
 
-## 🗺️ Roadmap
+## 📄 License
 
-See [ENHANCEMENT_PLAN.md](./ENHANCEMENT_PLAN.md) for the full technical roadmap.
+This project is licensed under the **MIT License**.
 
-### Coming Soon
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)